src/index.tsx

```diff
@@ -228,7 +228,10 @@ export class App extends React.Component<any, AppState> {
 
   public shouldComponentUpdate() {
     if (!history.isRecording()) {
-      this.componentDidUpdate();
+      // temporary hack to fix #592
+      setTimeout(() => {
+        this.componentDidUpdate();
+      });
```

Comment:
Nooo :(

Reply:
So, what do you want to do? This `this.componentDidUpdate` is a hack in itself :).